<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPackageMenu</class>
 <widget class="QWidget" name="AddPackageMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add package</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="pathToDeb"/>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="isDeb">
     <property name="text">
      <string>.deb</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="chooseFile">
     <property name="text">
      <string>Choose .deb-file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Package name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="aptPackageName"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QRadioButton" name="isApt">
     <property name="text">
      <string>From APT</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="isRequired">
       <property name="text">
        <string>Requierd</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
